frontend/src/main/scala/bloop/exec/JvmProcessForker.scala

```diff
@@ -112,21 +112,112 @@ final class JvmForker(config: JdkConfig, classpath: Array[AbsolutePath]) extends
       opts: CommonOptions,
       extraClasspath: Array[AbsolutePath]
   ): Task[Int] = {
+
+    def logDebugIfVerbose(debugMsg: => String): Task[Unit] =
+      if (logger.isVerbose) {
+        Task(logger.debug(debugMsg)(DebugFilter.All))
+      } else {
+        Task.unit
+      }
+
+    def runCmd(cmd: List[String]): Task[Int] = {
+      def debugOptions =
```

Comment:
@olafurpg Could you rebase on top of latest master and remove the debug logs? I removed these logs in another PR since they were redundant. The `Forker.run` expression and `logDebugIfVerbose` could now be inlined